cards/src/test/java/com/hiddenswitch/spellsource/SoulseekerTests.java

```diff
@@ -0,0 +1,291 @@
+package com.hiddenswitch.spellsource;
+
+import net.demilich.metastone.game.cards.Attribute;
+import net.demilich.metastone.game.cards.Card;
+import net.demilich.metastone.game.cards.CardCatalogue;
+import net.demilich.metastone.game.entities.minions.Minion;
+import net.demilich.metastone.tests.util.TestBase;
+import org.junit.Assert;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static org.testng.Assert.*;
+
+public class SoulseekerTests extends TestBase {
+
+	private static final List<String> SOULBIND_TOKENS = Arrays.asList(
```

Comment:
see, why not just use `SpecificCardFilter`, it even supports a `cards` argument now